README.md

```diff
@@ -0,0 +1,20 @@
+# multinet-docs
+
+This repository contains the source code for the Multinet project documentation.
+Read the live documentation at [Read The
+Docs](https://multinet-app.readthedocs.io), visit the [Multinet project page](),
```

Comment:
Is there a link for the project page? Is it the VDL one at: https://vdl.sci.utah.edu/projects/2019-nsf-multinet/ ?

Reply:
Yep, that's the one :joy:

I added the link in f572645bb20cc1db969ccb4984a61877985c5b92.